app/uk/gov/hmrc/eusubsidycompliancefrontend/config/AppConfig.scala

```diff
@@ -31,6 +31,9 @@ class AppConfig @Inject() (config: Configuration, contactFrontendConfig: Contact
   val euroOnlyEnabled: Boolean =
     config.getOptional[Boolean]("features.euro-only-enabled").getOrElse(false)
 
+  val xiEoriAdding: Boolean =
```

Comment:
sorry to be pedantic but can we rename the flag to `xi-eori-enabled` and `xiEoriEnabled` (i just think it reads a bit better). And we typically do `getOrElse(false)`. Lets stay consistent.

Reply:
its currently enabled so default is enabled in future we plan to disable
so would this read better
xiEoriAddingDisabled
and... features.xi-eori-adding-disabled

Reply:
updated